src/hooks/useCart/useCart.tsx

```diff
@@ -181,7 +220,8 @@ export const useCart = (
     getProduct,
     ids,
     prevIds,
-    products.length,
+    prevProducts,
+    products, // TODO: check with team any reason for using products.length instead of products
```

Comment:
I previously encountered some issues with this effect being ran too many times. But if using `usePrevious` and products doesn't change reference, then perhaps this works fine. Ultimately just need to make sure `getQuota` is only called when needed.

Reply:
ok got it